images/scorecard-proxy/cmd/proxy/main.go

```diff
@@ -43,6 +48,19 @@ func main() {
 			k8sutil.WatchNamespaceEnvVar)
 	}
 
+	port := defaultPort
+
+	var tmp string
+	tmp, found = os.LookupEnv("SCORECARD_PROXY_PORT")
+	if found {
+		i, err := strconv.Atoi(tmp)
+		if err != nil {
+			log.Fatal(err)
+		}
+		port = i
+	}
+	log.Infof("Listening on port %d.", port)
```

Comment:
Super minor nit and definitely non-blocking, but to me this implies that the server is up already, when in reality it gets launched further below. There's probably not much possibility of hanging or something between here and there, but maybe we should move this log message to below where the proxy is actually started, or reword it to make it clear that we will attempt to launch the proxy on this port later